LiteFarmOrg/LiteFarm
# Lf 2662 remove webhook column

Ticket: [LF-2662](https://lite-farm.atlassian.net/browse/LF-2662)

To test:
- run the latest migrations
- run `npm run ngrok`
- run `npm run ngrok:setup`
- then start both the frontend and backend servers
- Create a new user
- Add an ESCI sensor to the map
- Make sure the webhook_id is added correctly
